### ISSUE TITLE
[REG] what mse do you get from the mnist examples 

### ISSUE THREAD

**Issue by bhomass** [gugarosa/learnergy#38]
## Template when making a new issue

Please, make sure that the following boxes are checked before submitting a new issue. There is a small chance that you can solve it by your own or even that it was already addressed by someone.

Thank you!

### Pre-checkings

- [ x] Check that you are up-to-date with the master branch of Learnergy. You can update with:
pip install git+git://github.com/gugarosa/learnergy.git --upgrade --no-deps

- [ x] Check that you have read all of our [README](https://github.com/gugarosa/learnergy/blob/master/README.md).

### Description

[Description of the issue]

### Link
I ran through the example code in gaussian_rbm_training.py.
after 5 epochs, the mse is at 253.189896
then I followed up with reconstruction, and the mse jumps to 253349.031250, which is of course useless. 

I can increase the epoch counts, but for now,  I just want to make sure I am repeating what you see on your side. 

### Steps to Reproduce

1. [First Step] copy the code from gaussian_rbm_training.py, and run it inside a jupyter notebook
2. [Second Step] run thru training without any parameter change. get mse = 253.189896
3. [...] then run the reconstruction code as is, get  mse == 253349.031250,

**Expected behavior:** [It should be what you expect to happen]
mse value a lot closer to the training loop values

**Actual behavior:** [What actually happens]
mse is orders of magnitude worse on reconstruction

**Reproduces how often:** [How much does it occur? Show us your percentage]

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by gugarosa** [comment]
1. Different MSE computation context: During `fit()`, the MSE is computed against the reconstructions produced by CD-k Gibbs sampling (multiple alternating steps). During `reconstruct()`, it's a single forward-backward pass (hidden_sampling → visible_sampling). With only 5 epochs of training and 128 hidden units for 784-dimensional data, the learned weights aren't strong enough to produce accurate single-pass reconstructions.

2. `GaussianRBM` visible layer returns unbounded values: Unlike Bernoulli RBMs (which output sigmoid-clamped values in  [0,1]), `GaussianRBM.visible_sampling()` returns raw linear activations as states. Poorly learned weights can produce activations far from the input range, and squaring those errors amplifies the difference enormously.

3. Only 5 epochs is too few: RBMs typically need 50–100+ epochs for reasonable convergence, especially Gaussian variants which are harder to train than Bernoulli ones. The training MSE of 253 (across 784 pixels) is already high, that's ~0.32 squared error per pixel, which indicates the model hasn't converged.

4. Practical suggestions:
   - Increase epochs to 50+
   - Try a smaller learning rate (e.g., 0.001)
   - Increase hidden units (e.g., 256 or 512)
   - Consider using `VarianceGaussianRBM` which learns the variance parameter, or try the `GaussianReluRBM` variant
   - Remove the double normalization — the example applies `torchvision.transforms.Normalize((0.1307,), (0.3081,))` and the model's internal batch normalization (`self.normalize=True`), which may conflict
